controlplane/kubeadm/internal/webhooks/kubeadm_control_plane_test.go

```diff
@@ -912,20 +892,20 @@ func TestKubeadmControlPlaneValidateUpdate(t *testing.T) {
 			kcp:       featureGates,
 		},
 		{
```

Comment:
Happy to make some of these etcd fields immutable again. I just don't understand why it shouldn't be possible to mutate them.

Does anyone have more context? @fabriziopandini @vincepri maybe?

Reply:
no specific reason AFAIK. we just started with everything immutable and relaxed things slowly

Reply:
I understand that in general we added new mutable fields iteratively. I got confused that we explicitly had unit tests ensuring that these fields are immutable